META.json

```diff
@@ -0,0 +1,44 @@
+{
+   "name": "griddb_fdw",
+   "abstract": "Foreign Data Wrapper for GridDB databases",
+   "description": "PostgreSQL extension which implements a Foreign Data Wrapper (FDW) for GridDB.",
+   "version": "0.0.1",
```

Comment:
The latest version is 2.3.0.
So could you update this version?

Reply:
Updated.

Reply:
Thank you for support.